Source/WebCore/dom/Document.h

```diff
@@ -2540,6 +2543,7 @@ class Document
     RefPtr<ResizeObserver> m_resizeObserverForContainIntrinsicSize;
 
     const std::optional<FrameIdentifier> m_frameIdentifier;
+    bool m_cachedCookiesEnabled { true };
```

Comment:
Odds are this should be a std::optional<> so that we know if the cache is valid or not.